src/grpc/config/hpr_skf_ets.erl

```diff
@@ -0,0 +1,56 @@
+-module(hpr_skf_ets).
+
+-export([
+    init/0,
+    insert/1,
+    delete/1,
+    lookup_devaddr/1
+]).
+
+-define(ETS, hpr_skf_ets).
+
+-spec init() -> ok.
+init() ->
+    ?ETS = ets:new(?ETS, [public, named_table, set, {read_concurrency, true}]),
```

Comment:
Do we want to consider making this a table of `#config_session_key_filter_v1_pb{}` records?